backend/views/pusher_config.py

```diff
@@ -0,0 +1,23 @@
+from pusher import Pusher
+from django.http import JsonResponse
+from django.conf import settings
+import pusher
+from django.views.decorators.csrf import csrf_exempt
+
+
+pusher_client = Pusher(
+    app_id="1773977",
+    key="e44f77643020ff731b4f",
+    secret="064da109d46ae1fd75ee",
```

Comment:
If it is convenient, can we hide secret as a env variable? because it seems dangerous to expose this to github.